content/applications/finance/accounting/getting_started/main_concept.rst

```diff
@@ -131,8 +123,22 @@ average price, LIFO (for countries allowing it) and FIFO.
 Easy retained earnings
 ======================
 
-Retained earnings are the portion of income retained by your
-business. Odoo automatically calculates your current year earnings in
-real time so no year-end journal or rollover is required.  This is
-calculated by reporting the profit and loss balance to your balance
-sheet report automatically.
+Retained earnings are the portion of income retained by your business. Odoo automatically calculates
+your current year earnings in real-time, so no year-end journal or rollover is required. The profit
+and loss balance is automatically reported to your balance sheet report.
```

Comment:
```suggestion
Retained earnings are the portion of income retained by a business. Odoo calculates
current year earnings in real-time, so no year-end journal or rollover is required. The profit
and loss balance is automatically reported on the balance sheet report.
```
- "your"/"you" isn't necessary most of the time.
- Odoo calculates (automatically is implied here)
- Report on ≠ report to (i.e., your superior)